### PR TITLE
add MeshBlock in meshblock.py and refactor filenames for datablocks

### PR DESCRIPTION
**Pull request by alisterburt**

### CODE REVIEW

**Comment by alisterburt** [comment]
solves #16

File: peepingtom/core/datablocks/__init__.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
...
-from .properties import PropertyBlock
-from .spheres import SphereBlock
+from .imageblock import ImageBlock
+from .lineblock import LineBlock
+from .orientationblock import OrientationBlock
+from .particleblock import ParticleBlock
+from .pointblock import PointBlock
+from .propertyblock import PropertyBlock
+from .sphereblock import SphereBlock
+from .meshblock import MeshBlock
```
Comment (brisvag) [inline peepingtom/core/datablocks/__init__.py:8]: Missing newline! Fix that pycharm :P

File: peepingtom/core/datablocks/meshblock.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                array of indices into the vertices defining connectivity
+        kwargs : keyword arguments passed to DataBlock
+        """
+        super().__init__(**kwargs)
+        self.vertices = vertices
+        self.faces = faces
+        self.data = self.vertices
+
+    def _data_setter(self, vertices):
+        self._data = vertices
```
Comment (brisvag) [inline peepingtom/core/datablocks/meshblock.py:27]: I kee thinking that the we got ourselvess stuck with this `data` attribute and it's not necessary nor useful, especially in cases like this or `ParticleBlock` where it's not actually representing the whole data... We need to discuss about it!

File: peepingtom/core/datablocks/__init__.py (hunk 1 above)
Comment (brisvag) [inline peepingtom/core/datablocks/__init__.py:8]: These should also be changed everywhere else (depictors, peeper, tests...)

**Review by brisvag — CHANGES_REQUESTED** [review]

File: peepingtom/core/datablocks/__init__.py (hunk 1 above)
Comment (alisterburt) [reply peepingtom/core/datablocks/__init__.py:8]: everywhere else uses imports which derive from this as far as I can see! 
i.e.
`import peepingtom.core.datablocks.ImageBlock`
so this doesn't cause any problems

File: peepingtom/core/datablocks/meshblock.py (hunk 2 above)
Comment (alisterburt) [reply peepingtom/core/datablocks/meshblock.py:27]: we discussed this and agreed to drop it, not useful

**Pull request merged**